src/kata2.booleanToWord.js

```diff
@@ -1,3 +1,10 @@
-const booleanToWord = boolean => {};
+const booleanToWord = boolean => {
+    if (boolean === true) {
+        return "Yes";
+    } 
+    if (boolean === false) {
+        return "No"
+    }
+};
```

Comment:
return Boolean(boolean) ? "Yes" : "No"

It's a cleaner way. I'll always suggest ternary operators in cases where it's a 2-way choice!